packages/block-library/src/template-part/edit/index.js

```diff
@@ -232,13 +264,19 @@ export default function TemplatePartEdit( {
 		);
 	}
 
+	/*
+	 * PROTOTYPE HACK: Using different group props to create visual separation
+	 * between toolbar buttons. WordPress automatically adds separators between
+	 * different BlockControls groups. This avoids the need for custom CSS
+	 * while maintaining the standard toolbar appearance.
+	 */
```

Comment:
This is done elsewhere in the codebase and the groups don't assign any kind of meaningful descriptions or semantics, so this is fine to me. I don't know what the PROTOTYPE HACK is about though. Is this not meant to be merged?